### PR TITLE
Country Guide

### PR DESCRIPTION
**Pull request by AbhishekRP2002**
### 👨‍💻 Changes proposed
This project displays a list of common information which includes Population , currency , Country capital , Continent about that country including an image of the country’s flag.

### ✅ Check List (Check all the applicable boxes) 

- [x] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [x] This PR does not contain plagiarized content.
- [x] My Addition/Changes works properly and matches the overall repo pattern.
- [x] The title of my pull request is a short description of the requested changes.

### 📄 Note to reviewers

Decriptipon: Created a web app which displays a list of common information which includes Population , currency , Country capital , Continent about that country including an image of the country’s flag using RESTful API for fetching respective country data /information.
### 📷 Screenshots

cntryguide

### CODE REVIEW

**Comment by myhacktech** [comment]
@AbhishekRP2002 have u added a new webpage or exiting (then attach link)

**Comment by AbhishekRP2002** [comment]
I've added a new web page.

**Comment by myhacktech** [comment]
> I've added a new web page.

ok great

**Review by myhacktech — DISMISSED** [review]

**Comment by myhacktech** [comment]
@Astrodevil to ber merged

**Review by Astrodevil — COMMENTED** [review]
Follow alphabetical order in cards.json. Also add project in README. Don't put .vscode and don't make changes in style.css

**Comment by Astrodevil** [comment]
conflicts are there

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
+}
```
Comment (Anmol-Baranwal) [inline .vscode/settings.json:3]: Remove .vscode directory

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
Everything else working correctly !!

Please remove the .vscode folder then I will merge the PR.

**Pull request closed without merging**